### ISSUE TITLE
[Request]: Database Integration

### ISSUE THREAD

**Issue by markwillow** [GuavaCZ/filament-knowledge-base#44]
### What feature would you like to add?

I am currently working as a developer for a retail company. We are looking to enhance our knowledge base by transitioning from Markdown (MD) files to a database format. This migration will allow for more efficient data management and retrieval. I will be spearheading this project and providing feedback throughout the process to ensure a smooth transition and optimal implementation.

**Comment by lukas-frey** [comment]
Hi, a PR for that is welcome.

Each documentation file is already implemented like a sushi eloquent model, so it shouldn't be too hard to implement an additional real database model.

However, even then the documentation should be stored as markdown, to allow interoperability and to have to only maintain a single parser.

In my mind, I imagined the package would support three distinct way of writing / storing the knowledge base documentations:

1) Markdown files in /docs/ - current approach, already working
2) PHP classes stored in App/Docs/ - not working currently. In the background, these would be converted into sushi models as well
3) Database Eloquent models - not implemented yet. This would store the documentation in the database, allowing the documentation to be edited within the filament panels. Preferrably, this would also be implemented with a default editor. Kind of like a wikipedia page.

**Comment by markwillow** [comment]
That's right. However, I still need to review the MD documentation. I've been trying to use Blade View for this, but I noticed that it is not compatible, or maybe I'm doing it incorrectly.

In our case, we really need to have this converted to a database because that would be a lot of work for us developers to update stuff from time to time. Especially in emergency cases.

**Comment by AidanLaycock** [comment]
Hey, are you still looking for help with this? Or has some progress been made with it? As I'd be happy to give a helping hand where possible for it, as it would make this package even more awesome! 🎉

**Comment by lukas-frey** [comment]
Hello, yes, help is very welcome! There's a Database Model in the source already, but that's about all.

**Comment by Sjoerd24** [comment]
I just found this plugin and it is exactly what I needed, perfect! I would also love the database integration. Did you somewhere outline how you would make it and whats needs to be done?
